src/api/user.ts

```diff
@@ -0,0 +1,18 @@
+import { z } from 'zod'
+import { api } from '@/api/client'
+
+const userSchema = z.object({
+  id: z.number(),
+  lastName: z.string(),
+  firstName: z.string(),
+  createdAt: z.string().datetime(),
+  updatedAt: z.string().datetime(),
```

Comment:
nit) 문서 찾아보면 나오긴 하지만.. 그래도 어떤 표준 쓰는지 comment에 써놓으면 어떤가요?
```suggestion
  createdAt: z.string().datetime(), // ISO 8601
  updatedAt: z.string().datetime(), // ISO 8601
```

Reply:
코드를 보고 대략적인 이해를 하는 것이 낫다고 생각합니다.
앞으로 사용할 모든 `z.string().datetime()`에 코멘트를 달 수 없고 포멧이 달라질 수 있기 때문입니다.

FYI, MDN [Date time string format](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#date_time_string_format) & Zod [Datetimes](https://zod.dev/?id=datetimes)